drivers/watchdog/wdt_nrfx.c

```diff
@@ -21,41 +21,40 @@ struct wdt_nrfx_data {
 
 struct wdt_nrfx_config {
 	nrfx_wdt_t	  wdt;
-	nrfx_wdt_config_t config;
 };
 
 static int wdt_nrf_setup(const struct device *dev, uint8_t options)
 {
-	const struct wdt_nrfx_config *config = dev->config;
-	struct wdt_nrfx_data *data = dev->data;
 	nrfx_err_t err_code;
+	nrfx_wdt_config_t config;
```

Comment:
To make sure that all other fields in this structure are zeroed (just in case some fields are added in the future):
```suggestion
	nrfx_wdt_config_t config = {
		.reload_value = dev->data.m_timeout
	};
```